composer.json

```diff
@@ -131,7 +131,8 @@
     "bower-asset/select2-4.0.x": "~4.0.2",
     "bower-asset/slimScroll-1.3.x": "~1.3.7",
     "bower-asset/tinymce-builded": "^4.4",
-    "zeebinz/rarinfo": "~2.5.0"
+    "zeebinz/rarinfo": "~2.5.0",
+    "guzzlehttp/guzzle": "^6.3"
```

Comment:
I wasn't asking you to add it manually, just move it manually exactly as you asked me to do later.